docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`292` Add informatics param to LiquidHandle
```

Comment:
nit: Let's mention to `p.transfer` explicitly as well, since there are other helper methods for generating LiquidHandle instructions.